src/object/player.cpp

```diff
@@ -2415,6 +2415,7 @@ Player::collision(MovingObject& other, const CollisionHit& hit)
 
     if (hit.bottom && badguy->is_frozen())
       m_on_ground_flag = true;
+      m_physic.set_velocity_y(0);
```

Comment:
Is the `set_velocity_y` call *supposed* to be part of the if statement in line 2416 or not?

If it does, it needs braces after the if. If not, the indentation needs to get fixed or, even better, the call needs to be put in a new line.